lib/RTCSession.js

```diff
@@ -676,7 +676,17 @@ module.exports = class RTCSession extends EventEmitter
         this._localMediaStream = stream;
         if (stream)
         {
-          this._connection.addStream(stream);
+          if (this._connecting.addTrack)
```

Comment:
Have you tested this? I don't think `this._connecting.addTrack` is what you want...

Reply:
arf right! I miss my copy/paste

Reply:
fixed, sorry